### PR TITLE
update postgres image to be based on postgis/postgis:14-3.3-alpine

### PR DESCRIPTION
**Pull request by jcrides**
Updates to build a postgres image from postgis/postgis:14-3.3-alpine.

### CODE REVIEW

File: 00-setup-db.sh [hunk 1]
```diff
@@ -2,27 +2,14 @@
...
-EOS
-
 # If extensions are listed, create them
 echo
 for extension in $POSTGRES_EXTENSIONS; do
   psql \
     --echo-queries \
     --set ON_ERROR_STOP=1 \
     --username "$POSTGRES_USER" \
+    -d template1 \
```
Comment (noisybit) [inline 00-setup-db.sh:12]: What does this change do?

File: Dockerfile [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-    apt-get clean && \
-    apt-get autoremove
-
-COPY howgood-entrypoint.sh /
+# COPY howgood-entrypoint.sh /
 COPY 00-setup-db.sh /docker-entrypoint-initdb.d/
 
 # Default extensions
-ENV POSTGRES_EXTENSIONS hstore postgis postgis_topology pg_trgm fuzzystrmatch
+ENV POSTGRES_EXTENSIONS citext hstore postgis postgis_topology pg_trgm fuzzystrmatch
```
Comment (noisybit) [inline Dockerfile:9]: Were we always missing `citext`? Is it active in RDS?

File: Dockerfile [hunk 3]
```diff
@@ -1,23 +1,11 @@
...
-RUN apt-get update && \
-    apt-get install -y \
-          postgresql-$PG_MAJOR-postgis-$GIS_VERSION \
-          postgresql-contrib-$PG_MAJOR && \
-    rm -Rf /var/lib/apt/lists/* && \
-    apt-get clean && \
-    apt-get autoremove
-
-COPY howgood-entrypoint.sh /
+# COPY howgood-entrypoint.sh /
```
Comment (noisybit) [inline Dockerfile:5]: Can we just remove this line and the entrypoint wrapper completely if it is not needed?

File: Dockerfile (hunk 2 above)
Comment (jcrides) [reply Dockerfile:9]: It was probably already set up in the old base image we were using. I had to add it to get the new image to work.

File: Dockerfile (hunk 3 above)
Comment (jcrides) [reply Dockerfile]: Oops, missed this one. The file and line are removed now.

**Review by noisybit — APPROVED** [review]

**Pull request merged**